packages/site/src/types/banner.ts

```diff
@@ -0,0 +1 @@
+export type BannerType = "notice" | "success" | "warning" | "error";
```

Comment:
I'm approving this for now, but we'll need to figure out a better way for this! I think we can either pull the type in directly from Atlantis, or if that's not available we can export it from Atlantis! Either way, good to go for now!